### PR TITLE
Functions

### PR DESCRIPTION
**Pull request by kasparvonbeelen**

### CODE REVIEW

File: Sessions/1c-Intro-to-Python.ipynb [hunk 1]
```diff
@@ -291,66 +243,44 @@
...
     "\n",
-    "A common example in text mining is called text preprocessing, where we transform an input text to format that makes it more amenable to computational analysis. We discuss this in more detail later, but a simple preprocessing stap could be:\n",
+    "A common example in text mining is called text **preprocessing**, where we transform an input text into a format that makes it more amenable to computational analysis. We discuss this in more detail later, but a simple preprocessing step could be:\n",
     "\n",
-    "- Check if text is if minimal length 5\n",
-    "- if so, return the text in lowercased form\n",
-    "- Otherwise return and empty string"
+    "- Check if a text has more than 5 characters\n",
+    "- if so, return the text in the uppercased form (see the `.upper()` example in 1b)\n",
+    "- Otherwise, return an empty string"
```
Comment (mcollardanuy) [inline Sessions/1c-Intro-to-Python.ipynb]: Hi! We're doing string functions on Tuesday, so `upper()` does not appear until notebook 2a.

File: Sessions/1c-Intro-to-Python.ipynb [hunk 2]
```diff
@@ -380,23 +311,48 @@
...
-    "- The name used to call the function. You can use any name except those that belong the small category of Python keywords\n",
-    "- Parameters comprise the information you want to pass to a function\n",
-    "- Statemetns: the sequence of statements that perform an operation (on the input parameters).\n",
-    "- Function often and with a `return` statement"
+    "- The **name** is used to call the function. (You can use any name except those that belong the small category of Python keywords)\n",
+    "- **Parameters** comprise the information you want to pass to a function \n",
+    "    - i.e. What's the input?\n",
+    "- **Statements**: the sequence of statements that perform an operation\n",
+    "    - i.e. What will you do with the input?\n",
+    "- Function often end with a `return` statement\n",
```
Comment (mcollardanuy) [inline Sessions/1c-Intro-to-Python.ipynb]: Typo: "Functions often end"

**Review by mcollardanuy — DISMISSED** [review]
All good!

File: Sessions/1c-Intro-to-Python.ipynb (hunk 1 above)
Comment (kasparvonbeelen) [reply Sessions/1c-Intro-to-Python.ipynb]: Hmm... you're right. I thought there was one example. I'll adjust the text.

**Review by mcollardanuy — APPROVED** [review]

**Pull request merged**